Phases/Assessments/README.md

```diff
@@ -85,3 +88,14 @@ To prepare for this assessment, learners should consider the following questions
 - Why did you make this design decision?
 - What are the inputs and outputs?
 - How did you arrive at your solution? Walk through your commit history to show how your solution grew.
+
+### Membership Review
+
+After the evaluation & interview are completed, Membership will make a decision and inform the learner.
+
+In most cases, the decision will be either one of these two options:
+
+- **Pass**: learner continues on to the next phase
+- **Fail**: learner begins membership exit process
+
```

Comment:
Seems to me this section is written on the assumption that a learner is taking the assessment in the 8th week of the phase (i.e. the make or break point) - It should be re-written on the assumption that the learner is taking it in the 6th week (since this is the baseline expectation, with taking in the 8th week being an option for remediation)

I would re-write as follows:

###Membership Review

After the evaluation & interview are completed, Membership will make a decision and inform the learner.

The outcome will be one of these two options:

1. Pass: learner continues on to the next phase
2. Fail: depending on the learner's pace, a few different outcomes may follow:
--> 2a. if the learner failed the assessment before the 8th week of the phase, they will have until the 8th week to review, remediate, and re-take the assessment.
--> 2b. if the learner fails the assessment in their 8th week of the phase, they will begin the membership exit process
--> 2c. if the learner fails the assessment on their second attempt (regardless of the week) they will begin the membership exit process